azure-cli/README.md

```diff
@@ -0,0 +1,29 @@
+# GitHub Action for the Azure CLI
+
+The GitHub Action for [Azure CLI](https://github.com/Azure/azure-cli) wraps the Azure CLI to enable managing Azure resources inside of an Action.
+
+To log into a Azure, we recommend using the [Azure Login](../azure-login) Action.
+
+## Usage
+
+```
+
+ action "Azure CLI" {
+  uses = "Azure/github-actions/cli@master"
```

Comment:
This won't work. action is azure-cli. uses = "Azure/github-actions/azure-cli@master"

Reply:
@dikhakha 
Can you please update the name as per the spec.

Reply:
Can we make it consistent then? web app and functions deploy actions are also prefixed with azure-*. Will remove azure prefix for cli and login

Reply:
@vincentdass  @ds-ms  @nishubansal 
can you please update the names of your actions as well ( inline with what we have in documentation and in spec ). Repository name we can continue to use "Azure/github-actions"